parachain/pallets/inbound-queue/fixtures/src/register_asset.rs

```diff
@@ -1,21 +1,20 @@
 // Generated, do not edit!
 // See ethereum client README.md for instructions to generate
+
+use crate::InboundQueueFixture;
 use hex_literal::hex;
 use snowbridge_beacon_primitives::CompactExecutionHeader;
 use snowbridge_core::inbound::{Log, Message, Proof};
 use sp_std::vec;
 
-pub struct InboundQueueTest {
-	pub execution_header: CompactExecutionHeader,
-	pub message: Message,
-}
+pub type RegisterAsset = InboundQueueFixture;
```

Comment:
Nit: While I think "register asset" is probably more correct, everywhere else we refer to it as "register token" and so I think we should be consistent here and call it "RegisterToken".

Reply:
https://github.com/Snowfork/snowbridge/pull/1125/commits/eb5fb16f585dab6646955a54da9640a9556941fa